src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,69 @@
+package duke.util;
+
+/**
+ * Ui class deals with interactions with the user.
+ *
+ * @author Chng Zi Hao
+ */
+public class Ui {
+    private static final String DIVIDER = "----------------------------------"
+            + "----------------------------------------------";
+    private static final String PROMPT = "Enter Command: ";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String WELCOMEMESSAGE = String.format("%s\n%s\n%s\n%s", DIVIDER, LOGO,
```

Comment:
I think it will be nicer to name ```WELCOMEMESSAGE``` as ```WELCOME_MESSAGE```, spotted similar names somewhere else in this class as well